.rubocop.yml

```diff
@@ -31,9 +30,11 @@ Layout/EmptyLineAfterGuardClause:
   Enabled: false
 
 Layout/LineLength:
-  Max: 120
-  IgnoredPatterns: ['(\A|\s)#','.*\s+ # .*']
-  AllowHeredoc: true
+  # Temporarily disable while fixing sped/* directory
```

Comment:
I am a little concerned about disabling these checks (and the ones below), albeit hopefully only temporarily and for a short time, did you discuss with @jsugarman/did he have any suggestions? Do we know why we can't autogenerate the .rubocop_todo.yml?

Reply:
The trouble is that if I do not do it like this then for some reason (possibly a bug in Rubocop) it is not possible to auto-generate all the correct exceptions in `.rubocop_todo.yml` to get it to run without errors and warnings. I couldn't get to the bottom of what the issue was but I was trying to see if I could clean up enough to get it working by fixing the whitespace errors.

It should be simple to fix all the long lines but there are so many of them (nearly 1000) that I thought it would be better in a separate PR. Also, I think many of them could do with some more significant re-writing rather than simply breaking over multiple lines.

Reply:
👍